wasmcloud_basics/wash_intro/start.md

```diff
@@ -0,0 +1,36 @@
+## Starting wash
+
+`wash` can be run in two modes, interactive or single commands. In this tutorial we will explore the full power of `wash` in interactive mode. In order to start the `wash` REPL, we run `wash up`
+
+> Note: This is being run for you after the environment has fully bootstraped. This can take as long as 2 minutes.
+
+```bash
+❯ wash up --help
+wash-up 0.3.0
+Launch wasmcloud REPL environment
+
+USAGE:
+    wash up [OPTIONS]
+
+FLAGS:
+        --help    Prints help information
+
+OPTIONS:
+    -l, --log-level <log-level>    Log level verbosity, valid values are `error`, `warn`, `info`, `debug`, and `trace`
+                                   [default: debug]
+    -h, --host <rpc-host>          Host for lattice connections, defaults to 0.0.0.0 [env: WASH_RPC_HOST=]  [default:
+                                   0.0.0.0]
+    -p, --port <rpc-port>          Port for lattice connections, defaults to 4222 [env: WASH_RPC_PORT=]  [default:
+                                   4222]
+```
+
+### Navigating the UI
+
+There are four main sections of the UI. Starting at the top left is the REPL input box. This is where the user will enter their commands.
+`help`{{execute}}
+
+To the right of the input screen is the output section. This section shows the results from the command being executed. If the command results in an error, this displays in the TUI Log section.
```

Comment:
This last sentence should be "If the command results in any logs, those are displayed in the Tui Log section"